src/templates/error.js

```diff
@@ -1,13 +1,13 @@
+import ERROR_ICON from 'assets/SVG/playback-error.svg';
```

Comment:
This results in the error icon being added to jwplayer.js. We should not add the icon here. The only setup errors where we have loaded controls with icons would be an invalid playlist error. For that type of error we should add the icon in another part of the code (view.js or `core._view.somethingThatReturnsIconDOMElement`)